openarm_hardware/include/openarm_hardware/motor_control.hpp

```diff
@@ -1,47 +1,72 @@
-#ifndef MOTOR_CONTROL_H
-#define MOTOR_CONTROL_H
+#ifndef OPENARM_MOTOR_CONTROL_H_
+#define OPENARM_MOTOR_CONTROL_H_
 
 #include <iostream>
 #include <vector>
 #include <map>
 #include <array>
 #include <cstring>
 #include <unistd.h>
-#include "motor.hpp"
-#include "canbus.hpp"
+#include "openarm_hardware/motor.hpp"
+#include "openarm_hardware/canbus.hpp"
```

Comment:
We don't need this change because `#include "..."` searches from the same directory.
In this case, we have `motor.hpp` and `canbus.hpp` in the same directory as `motnor_control.hpp`. So `#include "motor.hpp"`/`#include "canbus.hpp` are better than `#include "openarm_hardware/motor.hpp"`/`#include "openarm_hardware/canbus.hpp"`.